CHANGELOG.md

```diff
@@ -462,6 +462,7 @@
 [@koic]: https://github.com/koic
 [@andyw8]: https://github.com/andyw8
 [@buehmann]: https://github.com/buehmann
+[@agrobbin]: https://github.com/agrobbin
```

Comment:
This will be added automatically during next version release.

Reply:
Ah! So should I I should remove it from the PR? Also I just realized I had *tried* to put it in the proper place alphabetically, but I totally screwed up.

Reply:
Yes, your name will be added automatically, [previous release](https://github.com/rubocop/rubocop-rails/commit/3552dd2cd385f3675a5a03ad33deba8e535473dd) for example.

Reply:
Removed!